assemble/conf/templates/accumulo-env.sh

```diff
@@ -25,6 +25,21 @@
 # Variables that must be set
 ############################
 
+## Accumulo logs directory. Referenced by logger config.
+export ACCUMULO_LOG_DIR="${ACCUMULO_LOG_DIR:-$ACCUMULO_HOME/logs}"
+## Hadoop installation
+export HADOOP_PREFIX="${HADOOP_PREFIX:-/path/to/hadoop}"
+## Hadoop configuration
+export HADOOP_CONF_DIR="${HADOOP_CONF_DIR:-${HADOOP_PREFIX}/etc/hadoop}"
+## Zookeeper installation
+export ZOOKEEPER_HOME="${ZOOKEEPER_HOME:-/path/to/zookeeper}"
+## See HADOOP-7154 and ACCUMULO-847
+export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
```

Comment:
Maybe not "must be set", but certainly strongly recommended, even for the average user. It's not a high-performance tweak; it's a workaround for bad memory management in glibc. That can impact average users, too. Not sure if this is still needed post-RHEL6, though.